cmd/siac/rentercmd.go

```diff
@@ -273,7 +273,8 @@ func renterallowancecmd() {
 
 // renterallowancecancelcmd cancels the current allowance.
 func renterallowancecancelcmd() {
-	fmt.Println("Canceling your allowance will destroy all contracts and uploaded files immediately.")
+	fmt.Println(`Canceling your allowance will prevent the renewal of your contracts and you will no longer be able to upload new files.
+	Already uploaded files can still be downloaded until the contracts expire and/or the file's redundancy drops below 1x.`)
```

Comment:
"Canceling your allowance will disable uploading new files, repairing existing files, and renewing existing files. All files will cease to be accessible after a short period of time."